### PR TITLE
Updated types to 1.7.43

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: project.yaml [hunk 1]
```diff
@@ -58,7 +58,14 @@ network:
         "tip": "Compact<Balance>",
         "target_asset_id": "AssetId"
     },
-    "ContentSource": "Vec<u8>",
+    "ContentSource": "Text",
```
Comment (pgolovkin) [inline project.yaml]: I suppose it should be Vec<u8>

File: project.yaml [hunk 2]
```diff
@@ -77,7 +84,7 @@ network:
         "default_fee": "BasisPoints",
         "default_protocol_fee": "BasisPoints"
     },
-    "Description": "Vec<u8>",
+    "Description": "Text",
```
Comment (pgolovkin) [inline project.yaml]: I suppose it should be Vec

**Review by pgolovkin — CHANGES_REQUESTED** [review]

**Comment by bgodlin** [comment]
Should types be regenerated within the [original repository](https://github.com/sora-xor/sora2-substrate-js-library/blob/master/packages/types/src/metadata/stage/types.json)?

File: project.yaml (hunk 2 above)
Comment (pgolovkin) [reply project.yaml]: No. It's a problem that we have. Because for some platforms we use Text, for others Vec<u8>

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**